src/test/java/com/quirko/logic/MatrixOperationsTest.java

```diff
@@ -402,4 +405,84 @@ public void testCheckRemoving4() {
         assertArrayEquals(expected, result.getNewMatrix());
     }
 
+    @Test
+    public void testCheckMerge() {

```

Comment:
Neyi test ediyorsun burada? Senin ekledigin degisikliklerle nasil bir iliskisi var?

Reply:
Hocam brick patlatma işlemini merge metodunun içinde yaptığım için bu testi yazma ihtiyacı duydum.